src/custom-element.js

```diff
@@ -1640,6 +1645,19 @@ function createBaseCustomElementClass(win) {
 
     /**
      * Returns an optional overflow element for this custom element.
+     * @return {!./layout-delay-meter.LayoutDelayMeter}
+     */
+    getLayoutDelayMeter_() {
+      if (!this.layoutDelayMeter_) {
```

Comment:
Doing this for every element might be expensive. Can we first start with one element, e.g. `amp-ad`? If you'd like, we could even configure which elements get this behavior using AMP_CONFIG.trackLayoutDelay or such?